Sources/StreamChat/Query/ChannelListQuery.swift

```diff
@@ -112,6 +112,23 @@ public extension FilterKey where Scope: AnyChannelListFilterScope {
     /// A filter key for matching the `blocked` value.
     /// Supported operators: `equal`
     static var blocked: FilterKey<Scope, Bool> { .init(rawValue: "blocked", keyPathString: #keyPath(ChannelDTO.isBlocked)) }
+    
+    /// A filter key for matching the `pinned` value.
```

Comment:
More complicated because filtered value needs to be a boolean, but the locally stored value is Date